<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContorClass</class>
 <widget class="QMainWindow" name="ContorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnDraw">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>绘图</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxContor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>等高线</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图像</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>73</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>层（Z）:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>205</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen_grid_file"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>计算</string>
    </property>
    <addaction name="actionGradientIndex"/>
    <addaction name="actionConformityIndex"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_grid_file">
   <property name="text">
    <string>open grid file</string>
   </property>
  </action>
  <action name="actionGradientIndex">
   <property name="text">
    <string>GradientIndex</string>
   </property>
  </action>
  <action name="actionConformityIndex">
   <property name="text">
    <string>ConformityIndex</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="contor.qrc"/>
 </resources>
 <connections/>
</ui>
